examples/ipam/main.tf

```diff
@@ -15,14 +15,14 @@ module "vpc" {
     }
     private = {
       netmask      = 24
-      route_to_nat = true
+      route_to_nat = "0.0.0.0/0"
     }
   }
 }
 
 #####################################
 # Example of a simple IPAM deployment
-# terraform apply -target=module.ipam_base_for_example_only
+# terraform apply -target=module.ipam_base_for_example_only -auto-approve
```

Comment:
revert this plz

Reply:
done, although this is just a helper command if someone wants to run the test from our example but nbd